runtime/opts/opts_linux.go

```diff
@@ -29,7 +29,7 @@ func WithNamespaceCgroupDeletion(ctx context.Context, i *namespaces.DeleteInfo)
 	if cgroups.Mode() == cgroups.Unified {
 		cg, err := cgroupsv2.LoadManager("/sys/fs/cgroup", i.Name)
 		if err != nil {
-			if err == cgroupsv2.ErrCgroupDeleted {
+			if err == cgroups.ErrCgroupDeleted {
```

Comment:
?

Reply:
because of #201

Reply:
```
v2: remove errors that are never returned
Unlike v1, the v2 package never returns these errors. 
```

Do we need this `if` check at all then?

Reply:
Good point. Seems it only returns ErrInvalidGroupPath

https://github.com/containerd/cgroups/blob/f8328fdc061d6fb757b370c19ad8f0d05a281a41/v2/manager.go#L199-L208

https://github.com/containerd/cgroups/blob/f8328fdc061d6fb757b370c19ad8f0d05a281a41/v2/paths.go#L49-L60

Reply:
That was the discussion on https://github.com/containerd/containerd/pull/5739

(sorry for short comment; typing from my phone)